plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -71,6 +73,7 @@ func (s *SchedulerServer) AddFlags(fs *pflag.FlagSet) {
 	s.ClientConfig.Burst = 100
 	client.BindClientConfigFlags(fs, &s.ClientConfig)
 	fs.StringVar(&s.AlgorithmProvider, "algorithm_provider", s.AlgorithmProvider, "The scheduling algorithm provider to use")
```

Comment:
So this line should look like:
```
fs.StringVar(&s.AlgorithmProvider, "algorithm_provider", s.AlgorithmProvider, "The scheduling algorithm provider to use, one of: " + ListAlgorithmProviders())
```

Where you're going to write the ```ListAlgorithmProviders()``` function. :)